### PR TITLE
(RORAB-2948) Removing the optimization of ROW to RANGE

### PR DESCRIPTION
**Pull request by Kowsikraja**

### CODE REVIEW

File: core/src/main/java/org/apache/calcite/rex/RexBuilder.java [hunk 1]
```diff
@@ -463,14 +463,6 @@ public RexWindow makeWindow(
       RexWindowBound lowerBound,
       RexWindowBound upperBound,
       boolean rows) {
-    if (lowerBound.isUnbounded() && lowerBound.isPreceding()
```
Comment (dm-shantanuc) [inline core/src/main/java/org/apache/calcite/rex/RexBuilder.java:466]: Missing test cases on Calcite end.

**Review by dm-shantanuc — CHANGES_REQUESTED** [review]

File: core/src/main/java/org/apache/calcite/rex/RexBuilder.java (hunk 1 above)
Comment (Kowsikraja) [reply core/src/main/java/org/apache/calcite/rex/RexBuilder.java:466]: Done

**Review by dm-shantanuc — CHANGES_REQUESTED** [review]

File: core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java [hunk 2]
```diff
@@ -11731,4 +11734,25 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
...
+    final RexNode analyticalFunCall = builder.getRexBuilder().makeOver(type,
+        SqlStdOperatorTable.MAX,
+        ImmutableList.of(), ImmutableList.of(), ImmutableList.of(orderKeys),
+        RexWindowBounds.UNBOUNDED_PRECEDING,
+        RexWindowBounds.UNBOUNDED_FOLLOWING,
+        true, true, false, false, false);
+    RelNode root = builder
+        .project(analyticalFunCall)
+        .build();
+    final String expectedOracleSql = "SELECT MAX() OVER (ORDER BY \"EMPNO\" ROWS BETWEEN "
```
[the remark this answers is not in the document]
Comment (Kowsikraja) [reply core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java:11753]: Yeah ok , i will get back to you once done

[the remark this answers is not in the document]
Comment (Kowsikraja) [reply core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java]: pushed the changes. please have a look

**Review by dm-shantanuc — APPROVED** [review]

**Review by dm-shantanuc — APPROVED** [review]
The changes look good.

**Pull request merged**